Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardSettingsViewController.swift

```diff
@@ -92,7 +92,10 @@ class CreditCardSettingsViewController: UIViewController, Themeable {
 
     func setupState() {
         // check if there are any starting config
-        guard startingConfig != nil else {
+        guard startingConfig == nil else {
```

Comment:
I think something went wrong with this change: the code is indented weirdly and `updateState(type: .empty)` is still at the bottom and there is an extra `}`. Like this builds are failing locally.